clientSide/theteddyproject/src/Components/Routes/Routes.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import './Routes.css'
+import Navbar from '../Navbar/Navbar'
+import Home from "../Home/Home";
+import Footer from '../Footer/Footer'
+
+
+function Routes() {
+  return (
+    <div className="App">
+     
+      <Navbar/>     
```

Comment:
please add routes in this file

Reply:
sorry I want to create the pages it was deleted thankyou